docs/develop/testing.rst

```diff
@@ -100,21 +100,20 @@ First time should take some time (it needs to create separate virtual
 environments and pull dependencies) but would ensure everything is fine.
 
 
-Travis CI
----------
+GitHub Actions
+--------------
 
-.. versionadded:: 1.0.4
+.. versionadded:: 2.4.0
 
-.. image:: https://secure.travis-ci.org/django-guardian/django-guardian.png?branch=master
-  :target: http://travis-ci.org/django-guardian/django-guardian
+.. image:: https://github.com/django-guardian/django-guardian/workflows/Tests/badge.svg?branch=devel
+  :target: https://github.com/django-guardian/django-guardian/actions/workflows/tests.yml
 
-Recently we have added support for Travis_, continuous integration server so it
-is even more easy to follow if test fails with new commits: http://travis-ci.org/#!/lukaszb/django-guardian.
+Recently we have added support for `GitHub Actions`_ so it is even more easy to follow
```

Comment:
```suggestion
We have support for `GitHub Actions`_ to make easy to follow
```

Documentation should be time independent to be maintainable. Nobody will remember to replace "Recently" with a better word after a few weeks.